.github/workflows/build-and-test.yaml

```diff
@@ -103,17 +92,6 @@ jobs:
         - otp: "24"
           elixir_version: "1.14"
 
-        - os: "ubuntu-18.04"
-          cc: "gcc-4.8"
-          cxx: "g++-4.8"
-          cflags: "-m32 -O2"
```

Comment:
Would it still be worth doing a 32-bit build (on a 64-bit architecture)?  We might find edge cases in scenarios like this, maybe?

Reply:
We still have -m32 for one target, however I still have in my plans to get proper support for something similar to Ubuntu 18.04, however the urgent change now is getting a reasonable CI working.